smrf/distribute/precipitation.py

```diff
@@ -164,6 +173,7 @@ def initialize(self, topo, data):
             """ model:  """.format(self.nasde_model))
 
         if self.nasde_model == 'marks2017':
+            self.thread_variables += ['storm_id']
```

Comment:
Mainly a style preference and I tend to use `.append()` for adding to an array. Works for numpy arrays, for instance, as well.

Reply:
Used append everywhere else but here...